pkg/server/events.go

```diff
@@ -144,8 +145,10 @@ func (em *eventMonitor) start() (<-chan struct{}, error) {
 					em.backOff.enBackOff(cID, evt)
 				}
 			case err := <-em.errCh:
-				logrus.WithError(err).Error("Failed to handle event stream")
-				close(closeCh)
+				if err != nil {
+					logrus.WithError(err).Errorf("Failed to handle event stream")
+					errCh <- err
```

Comment:
How do we know the difference between an bootup error and a normal runtime error?  Do we care?

Reply:
Since there is no grpc in between now, `Subscribe` either generates a real error https://github.com/containerd/containerd/blob/master/events/exchange/exchange.go#L153, https://github.com/containerd/containerd/blob/master/events/exchange/exchange.go#L194, or a error which should never happen https://github.com/containerd/containerd/blob/master/events/exchange/exchange.go#L178.